### DISCUSSION TITLE
The udp bandwith test

### DISCUSSION THREAD

**Discussion by terral001** [esnet/iperf#1454] [Q&A]
The server is win10  ,iperf3  3.1.3,run the command iperf3.exe -s
Th client is win11 ,the same version
when the clinent  iperf3.exe -c 192.168.68.145  -u -b 100m   it seems  lost 85%
![image](https://user-images.githubusercontent.com/51979828/211731398-05c1a787-6807-43bf-a884-76cd816ba442.png)
[  4]   0.00-10.00  sec   118 MBytes  99.1 Mbits/sec  0.678 ms  12729/14996 (85%)
when use the -R  ,the lost change    
C:\iperf>iperf3.exe -c 192.168.68.145 -b 100m -u -R
[  4]   0.00-10.00  sec   113 MBytes  95.2 Mbits/sec  0.141 ms  0/14518 (0%)

![image](https://user-images.githubusercontent.com/51979828/211731676-0531db4e-abd3-4a4b-bf85-c21674241cc9.png)

**Comment by davidBar-On** [comment]
@terral001, you are using a very old version of iperf3.  I am not sure whether this is the reason for the results you see, but it will help if you will use a newer version.  Newer iperf3 version for Windows is available [here](https://files.budman.pw/) (although not an official iperf3 site - maintained by [BudMan](https://www.neowin.net/forum/tags/iperf3/)).

Can you also send the output from the server side? (whether you will use a new iperf3 version or not.)

**Reply by bmah888** [reply]
In addition, providing some information about the test setup (what kind/speed of network links between the client and server?) might help also.

Bruce.
